exercises/concept/high-score-board/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+# Instructions
+
+In this exercise, you are implementing a way to keep track of the high scores for the most popular game in your local arcade hall.
+
+You have 6 functions to implement, mostly related to manipulating an object that holds the high scores.
+
+## 1. Create a new high score board
+
+Create a function `createScoreBoard` that creates an object that serves as high score board.
+The keys of this object will be the names of the players, the values will be their scores.
+For testing purposes you want to directly include one initial entry in the object.
+There should player `The Best Ever` with a score of `1000000` already on the board.
+
+```javascript
+createScoreBoard();
+// returns an object with one initial entry
+```
+
+## 2. Add players to a score board
+
+To add a player to the high score board, define the function `addPlayer`.
+It accepts 3 parameters:
+
+- The first parameter is an existing score board object.
+- The second parameter is the name of a player as a string.
+- The third parameter is the score as a number.
+
+The function returns the same score board object that was passed in after adding the new player.
+
+```javascript
+addPlayer({ 'Dave Thomas': 0 }, 'José Valim', 486373);
+// => {'Dave Thomas': 0, 'José Valim': 486373}
+```
+
+## 3. Remove players from a score board
+
+If a player did not visit the arcade hall for one year, they are removed from the high score board.
```

Comment:
Very minor/subjective:

This somewhat sounds like the task would be about implementing logic that detects old scores and removes them. It's not misleading since the rest of the task makes it clear what it's about but perhaps you could avoid this by changing it to something like "If a player was found to be cheating, they are manually removed from the high score board."

Reply:
I realized the same thing at some point but forgot to fix it then. Did it now.